app/controllers/api/authentications_controller.rb

```diff
@@ -1,4 +1,11 @@
 module Api
   class AuthenticationsController < BaseController
+    def edit_resource(type, id, data)
+      auth = resource_search(id, type, collection_class(:authentications))
+      task_id = auth.update_in_provider_queue(data)
+      action_result(true, "Updating Authentication with id #{id}", :task_id => task_id)
```

Comment:
hmm, I don't think we've ever had edits return action results. I'll need to test the api client gem.